server/src/main/kotlin/com/xebia/functional/xef/server/ai/providers/mlflow/PathProvider.kt

```diff
@@ -0,0 +1,68 @@
+package com.xebia.functional.xef.server.ai.providers.mlflow
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.plugins.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.auth.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import org.slf4j.Logger
+
+interface PathProvider {
+    fun chatPath(model: String, principal: UserIdPrincipal): String?
+    fun embeddingsPath(model: String, principal: UserIdPrincipal): String?
+}
+object DefaultPathProvider: PathProvider {
```

Comment:
What do we need this class for? It is not used

Reply:
Previous code, I'll remove it 👍🏼